packages/user-mgnt/src/features/verifyPassword/handler.ts

```diff
@@ -43,7 +43,11 @@ export default async function verifyPassHandler(
     // Don't return a 404 as this gives away that this user account exists
     throw unauthorized()
   }
-  if (generateHash(password, user.salt) !== user.passwordHash) {
+  if (!user.passwordHash) {
+    if (generateHash(password, user.salt) !== user.oldPasswordHash) {
+      throw unauthorized()
+    }
+  } else if (generateHash(password, user.salt) !== user.passwordHash) {
```

Comment:
Think we should add a comment here about why this is needed, and when we're going to remove it

Something in the lines of:
```
/*
 * In OCRVS-4979 we needed to change the hashing algoritm to conform latest security standards.
 * We still need to support users logging in with the old password hash to allow them to change their passwords to the new hash.
 *
 * TODO: In OpenCRVS 1.4, remove this check and force any users without new password hash to reset their password via sys admin.
 */
 if (generateHash(password, user.salt) !== user.oldPasswordHash) {
```